packages/vitest-pool-workers/test/helpers.ts

```diff
@@ -0,0 +1,158 @@
+/* eslint-disable no-empty-pattern */
+import childProcess from "node:child_process";
+import events from "node:events";
+import fs from "node:fs/promises";
+import path from "node:path";
+import util from "node:util";
+import { stripAnsi } from "miniflare";
+import { test as baseTest, inject, vi } from "vitest";
+
+const debuglog = util.debuglog("vitest-pool-workers:test");
+
+export const minimalVitestConfig = `
+import { defineWorkersConfig } from "@cloudflare/vitest-pool-workers/config";
+export default defineWorkersConfig({
+	test: {
+		testTimeout: 90_000,
+		poolOptions: {
+			workers: {
+				singleWorker: true,
+				miniflare: {
+					compatibilityDate: "2024-01-01",
+					compatibilityFlags: ["nodejs_compat"],
+				},
+			},
+		},
+	}
+});
+`;
+
+export function waitFor<T>(callback: Parameters<typeof vi.waitFor<T>>[0]) {
+	// The default timeout of `vi.waitFor()` is only 1s, which is a little
+	// short for some of these tests, especially on Windows.
+	return vi.waitFor(callback, { timeout: 10_000, interval: 500 });
```

Comment:
TIL! This would be pretty useful in lots of places

Reply:
New to me too...